src/colorizer/AnnotationData.ts

```diff
@@ -10,18 +10,32 @@ const CSV_COL_ID = "ID";
 const CSV_COL_TIME = "Frame";
 const CSV_COL_TRACK = "Track";
 
+export const BOOLEAN_VALUE = "TRUE";
+
 export const DEFAULT_ANNOTATION_LABEL_COLORS = KNOWN_CATEGORICAL_PALETTES.get(
   DEFAULT_CATEGORICAL_PALETTE_KEY
 )!.colorStops;
 
+export enum LabelType {
+  BOOLEAN = "boolean",
+  INTEGER = "integer",
+  CUSTOM = "custom",
+}
+
 export type LabelOptions = {
+  type: LabelType;
   name: string;
   color: Color;
+  autoIncrement: boolean;
```

Comment:
The label's type is part of its options, which affect how the label is displayed and interacted with. Internally, all labels use the same data model.

Reply:
I kinda feel like autoincrement should be a LabelType

Reply:
Updating here since we talked about this! I'm hesitant to make autoincrement a type for the specific case where we're importing a CSV as annotations back into the app. I can tell if a column is an integer or not, but I can't tell if it's intended to auto-increment.